### DISCUSSION TITLE
UNABLE TO ACCESS THE SOFTWARE THROUGH GITHUB LOG IN

### DISCUSSION THREAD

**Discussion by RukayatJ** [RosettaCommons/rosetta#193] [ROSIE] [answered]
Hy, I am trying to log into my GitHub account through the Rosetta Website. I have added and verified an institutional email address and i am still unable to access the software. My email address is rukayatjimoh@nimr.gov.ng ,the email is for the Nigerian Institute of Medical Research. What could be the problem please?
![image](https://github.com/user-attachments/assets/74cf5023-266d-4d0a-b177-239dce8abe86)

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! - this should be fixed now, - could you please give it another try? Please feel free to re-open this issue if you still having issues login-in. Thanks,
